tests/drivers/flash/performance/boards/tt_blackhole_tt_blackhole_smc.overlay

```diff
@@ -8,13 +8,11 @@
 &spi_flash {
 	status = "okay";
 	partitions {
-		compatible = "fixed-partitions";
-		#address-cells = <1>;
-		#size-cells = <1>;
-
 		storage_partition: partition@2008000 {
 			label = "storage";
 			reg = <0x2008000 DT_SIZE_K(32)>;
+			/* binary-path is a required property, but we don't care what it contains */
+			binary-path = "$BUILD_DIR/smc/generated_board_cfg/$BOARD_REV/read_only.bin";
```

Comment:
Could we make this property optional, and only generate a tt-boot-fs entry if the property is populated? Another option here might be to change where we set the `"tenstorrent,tt-boot-fs"` compatible to be on each child node.

Reply:
We can, but I would rather defer that to a separate PR, since this is really fixing a bug and that would be more of a feature (it would also involve a lot of hacks regarding `tt_boot_fs.py` for generating YAML from DT).

Reply:
@alexapostolu - can you create a ticket to make the `binary-path` property optional in the DT bindings and adjust `tt_boot_fs.py` accordingly?